modin/experimental/engines/omnisci_on_ray/frame/expr.py

```diff
@@ -121,37 +251,155 @@ def bin_op(self, other, op_name):
+        -------
+        BaseExpr
+            The resulting sub expression.
+        """
         return self.bin_op(other, "sub")
 
     def mul(self, other):
+        """
+        Build a mul expression.
+
+        Parameters
+        ----------
+        other : BaseExpr
+            The second operand.
+
+        Returns
+        -------
+        BaseExpr
+            The resulting mul expression.
+        """
         return self.bin_op(other, "mul")
 
     def mod(self, other):
+        """
+        Build a mod expression.
+
+        Parameters
+        ----------
+        other : BaseExpr
+            The second operand.
+
+        Returns
+        -------
+        BaseExpr
+            The resulting mod expression.
+        """
         return self.bin_op(other, "mod")
 
     def truediv(self, other):
+        """
+        Build a truediv expression.
+
+        The result always has float data type.
+
+        Parameters
+        ----------
+        other : BaseExpr
+            The second operand.
+
+        Returns
+        -------
+        BaseExpr
+            The resulting truediv expression.
+        """
         return self.bin_op(other, "truediv")
 
     def floordiv(self, other):
+        """
+        Build a floordiv expression.
+
+        The result always has an integer data type.
+
+        Parameters
+        ----------
+        other : BaseExpr
+            The second operand.
+
+        Returns
+        -------
+        BaseExpr
+            The resulting floordiv expression.
+        """
         return self.bin_op(other, "floordiv")
 
     def pow(self, other):
+        """
+        Build a power expression.
+
+        Parameters
+        ----------
+        other : BaseExpr
+            The power operand.
+
+        Returns
+        -------
+        BaseExpr
+            The resulting power expression.
+        """
         return self.bin_op(other, "pow")
 
     def floor(self):
+        """
+        Build a floor expression.
+
+        Returns
+        -------
+        BaseExpr
+            The resulting floor expression.
+        """
         return OpExpr("FLOOR", [self], get_dtype(int))
 
     def _cmp_op(self, other, op_name):
+        """
+        Build a comparison expression.
+
+        Parameters
+        ----------
+        other : BaseExpr
+            A value to compare with.
+        op_name : str
+            The comparison operation name.
+
+        Returns
+        -------
+        BaseExpr
+            The resulting comparison expression.
+        """
         lhs_dtype_class = self._get_dtype_cmp_class(self._dtype)
         rhs_dtype_class = self._get_dtype_cmp_class(other._dtype)
         res_dtype = get_dtype(bool)
         # In OmniSci comparison with NULL always results in NULL,
         # but in Pandas it is True for 'ne' comparison and False
         # for others.
-        # Also Pandas allow 'eq' and 'ne' comparison for values
+        # Also Pandas allows 'eq' and 'ne' comparison for values
```

Comment:
According to [internal convention](https://github.com/modin-project/modin/issues/2934#issuecomment-815494134) lowercase `pandas` should be used.

Reply:
Fixed